### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by CharlesAMiller**
Adding an initial outline for a Getting Started guide.

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* [Fizzi Teaches Basic Assembly and Melee Modding](https://www.youtube.com/watch?v=NOq49h0tkBI) (video) - Recorded by the creator of Slippi, this video teaches some basic assembly and demonstrates EXI communication.
+* [SSBM Data Sheet](https://docs.google.com/spreadsheets/d/1JX2w-r2fuvWuNgGb6D3Cs4wHQKLFegZe2jhbBuIhCG8/preview#gid=12) - A Google Sheet containing information about Melee's memory addresses.
+* [Assembler Tutorial](http://wiibrew.org/wiki/Assembler_Tutorial) - A wii modding specific guide to PPC, and its instructions. 
+* [Intro to Wii Game Modding by InternetExplorer](https://www.youtube.com/watch?v=IOyQhK2OCs0&list=PL6GfYYW69Pa2L8ZuT5lGrJoC8wOWvbIQv) (video) - Recorded by Dan Salvato - A playlist of videos related to modding Wii games. An excellent resource for debugging with Dolphin.
+
+## Tools
+
+### Melee Modding and Assembly
+* [HxD](https://mh-nexus.de/en/hxd/) - A free Hex editor. Hex editors in general are useful for looking through memory dumps. 
+* [SpeedCrunch](https://speedcrunch.org/) - A calculator for programmers. It allows for quick conversions between hex, binary, octal, and decimal.
```
Comment (NikhilNarayana) [inline GETTING_STARTED.md]: Windows' standard calculator also supports this btw.

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Via EXI communication. An example of such is demonstrated in [this video](https://github.com/project-slippi/Ishiiruka/search?q=rollback&unscoped_q=rollback) by Fizzi.
+
+> "Where is the matchmaking code?"
+
+The source code for the matchmaking server is in a private repository, and is not currently planned to be open sourced.
+
+## Guides & Resources
+
+### Melee Modding and Assembly
```
Comment (NikhilNarayana) [inline GETTING_STARTED.md:51]: maybe add a link to the IBM PPC reference sheet

File: GETTING_STARTED.md (hunk 1 above)
Comment (CharlesAMiller) [reply GETTING_STARTED.md]: > 
> 
> Windows' standard calculator also supports this btw.

I guess more notably, you can perform operations on numbers of different bases. I find it pretty useful for adding decimal offsets, to memory addresses - which are in hex. It also has configurable constants, so it's been useful to have pre-defined memory locations that I can reference without having to copy and paste. It's also available on macOS and Linux.

File: GETTING_STARTED.md (hunk 2 above)
Comment (CharlesAMiller) [reply GETTING_STARTED.md:51]: Just updated. I think I included the reference sheet you're referring to. In any case, I found it pretty informative.

Comment (NikhilNarayana) [reply GETTING_STARTED.md:51]: I was thinking of this one:

https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/assembler/idalangref_ins_set.html

**Pull request merged**